IFC4x3/Properties/c/CountOfUnits_2St0bSvg5E2uk2FhEnvtDZ/Documentation.md

```diff
@@ -0,0 +1 @@
+Number of retarders / [retarder(s)].
```

Comment:
Not sure what this format "retarders / [retarder(s)]" means.

Reply:
Pset erased.

Reply:
property erased too 29536c8c82c47bff18f29dc38343d8b2a1d20c9c